CMakeLists.txt

```diff
@@ -270,7 +270,11 @@ else()
   if(WIN32)
     add_compile_flag("-D_GNU_SOURCE")
     add_compile_flag("-D__STDC_FORMAT_MACROS")
-    add_link_flag("-Wl,--stack,8388608")
+    if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+      add_link_flag("-Wl,/stack:8388608")
```

Comment:
I think this will work in both cases won't it?  No need for an if here.

Reply:
This was originally a MinGW path, right? `/stack` would be interpreted as a file path on GCC.

Reply:
I'm not sure.. I'm pretty sure `--stack` is not a valid GNU ld linker flag.   What linker to you think it was meant for?

Reply:
Hmm.. seems you are right.. it looks like the MinGW version of ld does tack `--stack`